stripe/src/main/java/com/stripe/android/CustomerSession.java

```diff
@@ -82,6 +88,8 @@
             new HashMap<>();
     @NonNull private final Map<String, SourceRetrievalListener> mSourceRetrievalListeners =
             new HashMap<>();
+    @NonNull private final Map<String, PaymentMethodRetrievalListener>
```

Comment:
I wonder if we should have a single Map for all the in-flight listeners and just cast accordingly? Not necessary for this patch but something to consider.